package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "metrics-automation-reporter",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lucasbald/metrics-automation-reporter.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/lucasbald/metrics-automation-reporter/issues"
+  },
+  "homepage": "https://github.com/lucasbald/metrics-automation-reporter#readme",
```

Comment:
We can include engines and pre-commit:
```suggestion
  "homepage": "https://github.com/lucasbald/metrics-automation-reporter#readme",
  "engines": {
    "node": "16.x"
  },
  "pre-commit": [
	"audit",
	"lint"
  ],
```

Reply:
added precommit package as well
![image](https://user-images.githubusercontent.com/4924920/200316167-f596238e-95d4-457a-8290-732035f28ed4.png)